### PR TITLE
Jest-Axe(a11y) Tests

### PR DESCRIPTION
**Pull request by ElvisJG**
Todo
- baseURL needs to be set by environment variables in the CI/CD pipeline to add it to the current pipeline
- Manual a11y/508 compliance testing (Scheduled tomorrow using Wave and Axe)

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -18,22 +18,23 @@ const Wrapper = styled.div`
...
-  console.log("location ", location.pathname);
   return (
     <Wrapper className="display-flex flex-column">
       <Header basic color="primary" role="banner">
         <Banner />
         <div className="display-flex flex-justify-center border-bottom border-base-lighter">
           <div className="padding-top-2 padding-bottom-1 height-auto">
             <Grid row className="flex-align-center ">
-              <a href="/" title="Home" aria-label="Home">
+              <a href="/" title="Home" aria-label="Home" data-testid="homeLink">
```
Comment (jlhogan) [inline src/components/Layout.tsx:28]: if these changes were all for the e2e tests (adding data-testids) maybe better to keep them out of this PR?

Comment (ElvisJG) [reply src/components/Layout.tsx]: removed and added the changes to fix the depths issue on GuidedSearch

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
```
Comment (jlhogan) [inline jest.config.js:1]: let's remove this file! since jest config happens in package.json this will just create confusion.

File: package.json [hunk 3]
```diff
@@ -28,14 +30,15 @@
...
+    "ts-jest": "^28.0.7",
     "ts-node": "~10.8.1",
     "typescript": "~4.7.3",
     "web-vitals": "~2.1.4"
   },
   "scripts": {
     "start": "ts-node src/data/process_data.ts && react-scripts start",
     "build": "ts-node src/data/process_data.ts && react-scripts build",
-    "test": "ts-node src/data/process_data.ts && react-scripts test",
+    "test": "react-scripts test",
```
Comment (jlhogan) [inline package.json:41]: undo this change now that the data is used in the tests explicitly, we want to make sure it's always created & available & in the correct format!

Comment (ElvisJG) [reply package.json]: done 👍

File: jest.config.js (hunk 2 above)
Comment (ElvisJG) [reply jest.config.js]: done 👍

**Review by jlhogan — APPROVED** [review]
